packages/dashboards/addon/templates/components/navi-dashboard.hbs

```diff
@@ -100,6 +100,7 @@
         classNames="add-widget primary action"
         reports=(get (get-user) "reports")
         dashboard=dashboard
+        onAddWidgetToDashboard=(action onAddWidgetToDashboard)
```

Comment:
Should this be `onSelectedReport`?

Reply:
It's used as the click action handler of the "Add to dashboard" button, so `onAddWidgetToDashboard` is more specific and appropriate imo. The `onchange` of the select only mutates the `selectedReport` property internally.

Reply:
To me `onAddWidgetToDashboard` sounds like the widget was already added to the dashboard, when really the only event that happened was that a report was selected **to be added**.

`onSelect`?

Reply:
Oh I see. `onSelect` sounds like an action handler for the select's `onchange`, but I don't have a strong objection here. Let's go with your first suggestion `onSelectedReport`

Reply:
updated